qa/rpc-tests/test_framework/authproxy.py

```diff
@@ -67,7 +67,18 @@ def EncodeDecimal(o):
 class AuthServiceProxy(object):
```

Comment:
Good suggestion. I've started working on the implementation of this, and I think it's going to simplify things a bit.